backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/types/AcceptingStore.kt

```diff
@@ -0,0 +1,20 @@
+package app.ehrenamtskarte.backend.stores.importer.types
+
+data class AcceptingStore(
+    val name: String,
+    val countryCode: String,
+    val location: String,
+    val postalCode: String?,
+    val street: String?,
```

Comment:
Do we allow street to be null? 🤔  Does this occur?

Reply:
Yes, e.g. for mountain huts or natural baths.
> 'Erholungsgelände Ambach, Münsing' has no street set.
'Freibad Bichl, Bichl' has no street set.
'Freibad Ebensfeld (Badesee), Ebensfeld' has no street set.
'Gundischhof, Bad Wiessee' has no street set.
'Neunkirchener Konzerte, Neunkirchen am Brand' has no street set.
'Stie-Alm, Lenggries' has no street set.
'Waldnaabtal Ost/West, Windischeschenbach' has no street set.